.readthedocs.yml

```diff
@@ -4,12 +4,16 @@
 
 version: 2
 
+build:
+  os: ubuntu-22.04
```

Comment:
Nitpick: I would prefer ubuntu-latest so we don't have to update this line anymore